aml-development/frontend
# Live notifications

Implement real-time notification updates using websockets

Acceptance Criteria:
The user will receive close to real time notification updates on the site. (ie: if a user receives a new notification, the notification bell will light up, or increment numbers, without the need for the user to refresh the page)

How to test:
Create a new notification on old center (2.0) and you will see the notification bell increment without refreshing the page.
